cmdline/diff.py

```diff
@@ -0,0 +1,91 @@
+#!python
```

Comment:
it seems that you have again managed to get confused/or confuse git ;)  `cmdline` should be a submodule under nibabel module, not top level. So all the "fixes" below alike replacing `from nibabel.cmdline` to `from cmdline` are incorrect